src/unstable/components/Form/FieldTemplate.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { Box } from '../../../components/Grid';
+
+const LabelElement = styled.label`
+	font-size: 11px;
+	color: #252629;
+	text-transform: uppercase;
+	display: inline-block;
+	margin-bottom: 6px;
+`;
+
+const REQUIRED_FIELD_SYMBOL = '*';
+
+const Label = (props: any) => {
+	const { label, required, id } = props;
+	if (!label) {
+		return null;
+	}
+	return (
+		<LabelElement className="control-label" htmlFor={id}>
+			{label}
+			{required && <span className="required">{REQUIRED_FIELD_SYMBOL}</span>}
+		</LabelElement>
+	);
+};
+
+const FieldTemplate = (props: any) => {
+	const {
+		id,
+		classNames,
+		label,
+		children,
+		errors,
+		help,
+		description,
+		hidden = false,
+		required = false,
+		displayLabel = true,
+	} = props;
+	if (hidden) {
+		return children;
```

Comment:
I would expect that `hidden` would mean `return null` :confused: 
Would using something like `noLabel` make more sense (or maybe I'm missing something).

Reply:
These are the props passed to the component by `react-jsonschema-form`, so we can't change them. I actually lifted this component from the module's source  https://github.com/mozilla-services/react-jsonschema-form/blob/master/src/components/fields/SchemaField.js#L97